Admin.js

```diff
@@ -1,25 +1,26 @@
 /*
-
- This file is part of the Sonata package.
-
- (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
-
- For the full copyright and license information, please view the LICENSE
- file that was distributed with this source code.
-
+ * 
+ * This file is part of the Sonata package.
+ * 
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ * 
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ * 
  */
 
 var Admin = {
 
-    lolCounters: [],
+    "lolCounters": [],
 
     /**
      * This function must be called when an ajax call is done, to ensure
      * the retrieved html is properly setup
      *
      * @param subject
      */
-    shared_setup: function(subject) {
+    "shared_setup" (subject) {
```

Comment:
- eslint: Method 'shared_setup' has too many statements (13). Maximum allowed is 10.